.docker/readme.md

```diff
@@ -0,0 +1,55 @@
+If you'd like to only setup WordCamp.org and would like to use Docker, you can use inbuilt docker provisioning. Please follow these steps
+
+**Note:** This will create `.docker/database` directory which will contain MySQL files to persist data across docker restarts.
+
+1. Make sure you have Docker installed and Docker daemon running on your system.
+
+1. Clone the repo: 
+    ```
+    git clone git@github.com:WordPress/wordcamp.org.git
+    ```
+
+1. Change into the directory, and run docker compose command. This could take some time depending upon the internet speed.
+    ```
+    cd wordcamp.org && docker-compose up --build
+    ```
+     This will also start the docker environment for WordCamp.org development.
+     
+1. Add the following resolutions to your host file:
+    ```
+    127.0.0.1	central.wordcamp.test
+    127.0.0.1	wordcamp.test
+    127.0.0.1	2014.content.wordcamp.test
+    127.0.0.1	2014.misc.wordcamp.test
+    127.0.0.1	2014.new-site.wordcamp.test
+    ```
+    
+    `/wp-admin` pages for these sites should now be accessible. Use `admin` as username and `password` as password to login.
+    
+    **Note:** `https` URL scheme must be used to visit these sites. Security exception will be required in first time run.
+    
+1. Basic installation comes without themes and with only very basic plugins. Add plugins and themes which are installed in wordcamp.org if needed like so:
+
+    1. Find the docker which is running WordCamp. It will start with `wordcamporg_wordcamp.test` and will most likely be `wordcamporg_wordcamp.test_1`
```

Comment:
Changed the default scripts in https://github.com/WordPress/wordcamp.org/pull/113/commits/ac11b763700e157d761baedda4253bf3722d6e60. 

This can be done by using `docker-compose exec wordcamp.test bash`

Reply:
Nice! That's much better.